src/lib/y2security/clients/security_policy_proposal.rb

```diff
@@ -128,17 +128,30 @@ def links
         links.flatten.compact.uniq
       end
 
+      # Returns the warning message
+      #
+      # @return [String,nil] warning message or nil if there are no failing rules
+      def warning
+        return nil if success?
+
+        _("The current configuration does not comply with the enabled security policies")
```

Comment:
The language is fine.

What about a period `.` at the end? Do other messages in this context include it?

Reply:
You are right: e.g., we are using a period in the software proposal. I have added it to the message.